### ISSUE TITLE
Error if using Node 7.1

### ISSUE THREAD

**Issue by sebastianovide** [SitePen/remap-istanbul#103]
```
path.js:7
    throw new TypeError('Path must be a string. Received ' + inspect(path));
```

Any plans for supporting Node >= 7 ?

**Comment by dylans** [comment]
I'm always reluctant to work around the odd version numbers of Node.js given that they are not considered stable. That said, if it is a breaking API change, we'll of course be happy to address it and/or review a PR.

**Comment by sirrocco** [comment]
Hi,

This is probably the issue that happens with node 6 and that is fixed in the commit 0ff2b41e09bdb9b531df534bd266a8f0ea020d69 . It just needs to be released.

**Comment by dylans** [comment]
This should be fixed with 0.8.0, https://github.com/SitePen/remap-istanbul/releases/tag/0.8.0. Please re-open if not, thanks!
